core/scripts.py

```diff
@@ -2905,45 +2899,113 @@ def inventory_report_query(self) -> str:
+         order by title, author
+        """
+
+    def holds_report_query(self) -> str:
+        return """
+            select
+               e.title,
+               e.author,
+               i.identifier,
+               e.language,
+               e.publisher,
+               e.medium as format,
+               w.audience,
+               wg.genres,
+               d.name data_source,
+               ic.name collection_name,
+               coalesce(lib_holds.active_hold_count, 0) library_active_hold_count,
+               CASE WHEN collection_sharing.is_shared_collection THEN lp.patrons_in_hold_queue
+                    ELSE -1
+               END shared_active_hold_count
+        from datasources d,
+             collections c,
+             integration_configurations ic,
+             integration_library_configurations il,
+             libraries lib,
+             works w left outer join (select wg.work_id, string_agg(g.name, ',' order by g.name) as genres
+                                     from genres g,
+                                     workgenres wg
+                                     where g.id = wg.genre_id
+                                     group by wg.work_id) wg on w.id = wg.work_id,
+             editions e,
+             (select lp.presentation_edition_id,
+                                         p.library_id,
+                                         count(h.id) active_hold_count
+                                  from holds h,
+                                       licensepools lp,
+                                       patrons p
+                                  where p.id = h.patron_id and
+                                        h.license_pool_id = lp.id and
+                                        p.library_id = :library_id
+                                  group by p.library_id, lp.presentation_edition_id) lib_holds,
+             identifiers i,
+             (select ilc.parent_id,
+                      count(ilc.parent_id) > 1 is_shared_collection
+              from integration_library_configurations ilc,
+                   integration_configurations i,
+                   collections c
+              where c.integration_configuration_id = i.id  and
+                    i.id = ilc.parent_id group by ilc.parent_id) collection_sharing,
+             licensepools lp
         where lp.identifier_id = i.id and
               e.primary_identifier_id = i.id and
+              e.id = lib_holds.presentation_edition_id and
+              e.id = w.presentation_edition_id and
               d.id = e.data_source_id and
               c.id = lp.collection_id and
               c.integration_configuration_id = ic.id and
               ic.id = il.parent_id and
               ic.id = collection_sharing.parent_id and
+              ic.goal = 'LICENSE_GOAL' and
+              (ic.settings -> 'include_in_inventory_report' = 'true' or
+              ic.settings -> 'include_in_inventory_report' is null) and
```

Comment:
I don't think this is going to filter for what you want. Since you are just checking if this key isn't set, won't this select all of the collections in a CM, not just the collections we are interested in.